lobby_server.yaml

```diff
@@ -0,0 +1,16 @@
+- version: 1.8.0.9
```

Comment:
To be clear, this file can be updated with additional elements, so we can define many host/port/message+version combinations. Right now we only have the one entry.